sup3r/preprocessing/exogenous_data_handling.py

```diff
@@ -35,24 +36,19 @@ def __init__(self, file_paths, features, source_h5, s_enhancements,
         s_enhancements : list
             List of factors by which the Sup3rGan model will enhance the
             spatial dimensions of low resolution data from file_paths input.
-            For example, if file_paths has 100km data and s_enhance is 4, this
-            class will output a feature raster corresponding to the file_paths
-            grid enhanced 4x to ~25km. The length of this list should be equal
-            to the number of model steps. e.g. if using a model with 2 5x
-            spatial enhancement steps and a single temporal enhancement step
-            s_enhancements should be [5, 5, 1]
+            For example, if file_paths has 100km data and there are 2x spatial
```

Comment:
*2 spatial

Reply:
This should prob also mention that [1, 4] assumes that any temporal enhancement following the 5x spatial doesn't take exo data, otherwise this should be [1, 4, 5]